Assets/Scripts/Resources/Ironcount.cs

```diff
@@ -5,24 +5,38 @@
 
 public class Ironcount : MonoBehaviour
 {
-    public Text mytext = null;
-    private static int currentiron = 99;
+    public Text display = null;
+    private static int currentIron = 99;
+    private int i = 0;
+    private static int growth = 1;
 
-    void Awake()
+    void Start()
```

Comment:
```suggestion
    void Awake()
```
Awake() tutaj też albo reszte zmień na Start(), bez różnicy.